Sources/TuistLoader/Loaders/CachedManifestLoader.swift

```diff
@@ -191,6 +196,28 @@ public class CachedManifestLoader: ManifestLoading {
         return hash
     }
 
+    private func calculatePluginsHash(at path: AbsolutePath) throws -> String? {
+        guard let plugins = plugins else {
+            return nil
+        }
+
+        if let cached = pluginsCache[path] {
+            return cached
+        }
+
+        let projectDescriptionHelpersHash = try plugins.projectDescriptionHelpers
+            .map { try projectDescriptionHelpersHasher.hash(helpersDirectory: $0.path) }
+            .joined(separator: "-")
+            .md5
+
+        let pluginsHash = [
```

Comment:
Left this here in case we need to combine hashes of more than the helpers. Maybe for things like `Tasks`? @fortmarek